pkg/scm/git/git.go

```diff
@@ -34,7 +34,7 @@ type stiGit struct {
 	runner util.CommandRunner
 }
 
-var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/]+\.git)$`)
+var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/])`)
```

Comment:
Note on [custom port](http://stackoverflow.com/questions/5767850/git-on-custom-ssh-port), the commentary there explicitly states that you can only specify a port *IF* you specify the protocol (man git-clone also implies this).  As such, golang's url.Parse handles the strings where the protocol is specified, so we should be good there.